services/github/github-actions-workflow-status.service.js

```diff
@@ -8,8 +8,23 @@ const schema = Joi.object({
   workflow_runs: Joi.array()
     .items(
       Joi.object({
+        status: Joi.equal(
+          'completed',
+          'action_required',
+          'cancelled',
+          'failure',
+          'neutral',
+          'skipped',
+          'stale',
+          'success',
+          'timed_out',
+          'in_progress',
+          'queued',
+          'requested',
+          'waiting'
+        ).required(),
```

Comment:
Values for status are taken from

https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#get-a-workflow-run

![Screenshot at 2022-12-15 20-09-48](https://user-images.githubusercontent.com/6025893/207958481-950747e1-4a80-4e04-ad33-313d60d3b1c4.png)

I was in two minds about whether to add all the status values to `build-status.js` and attempt to classify them. The solution I've settled on is to leave that as-is and if we fall back to a status value and we don't have a formatting rule for it, just use default lightgrey for formatting.

I was also in two minds about whether to use a closed list here or allow any string (but make it required). Open to feedback on it..